front/app/components/SignUpIn/SignUp/PasswordSignup.tsx

```diff
@@ -531,7 +529,6 @@ class PasswordSignup extends PureComponent<Props & InjectedIntlProps, State> {
                   id="token"
                   type="text"
                   value={token}
-                  placeholder={formatMessage(messages.tokenPlaceholder)}
```

Comment:
Why remove all placeholders?

Reply:
Because they were removed in Crowdin (just empty strings) since we have proper form labels that have the same text the placeholders had. You can see it in the en.json file. Pretty good way to confuse developers :-)